webui/react/src/stores/users.tsx

```diff
@@ -87,47 +74,46 @@ class UserStore extends PollingStore {
     return () => canceler.abort();
   }
 
-  public fetchUsers(params: GetUsersParams = {}, signal?: AbortSignal) {
```

Comment:
question: `params` is not used now or in the future?

Reply:
For the store I was thinking we won't be need it in the foreseeable future, just because we are getting the full list upfront and polling too.